apiserver/upgradeservice/upgradeimpl.go

```diff
@@ -26,11 +32,16 @@ import (
 	"k8s.io/apimachinery/pkg/labels"
 )
 
-// CreateUpgrade ...
-func CreateUpgrade(request *msgs.CreateUpgradeRequest, ns string) msgs.CreateUpgradeResponse {
+//Currently supported version information for upgrades
+const REQUIRED_MAJOR_PGO_VERSION = "4"
```

Comment:
Should this be an `int` and not a string like `MAXIMUM_MINOR_PGO_VERSION` and `MINIMUM_MINOR_PGO_VERSION`?

Reply:
For the major version, the string value is correct. I apologize for confusion on this one, as I debated a bit. The initially grabbed value for both major and minor version returns as a string. In the case of the minor versions, I'm doing less than/greater than comparisons, so conversion to an int was needed. For the Major version, though, it can only be "4", so a string comparison was valid. All that to say, the code works as expected as is.

Obviously there are other ways to handle this to make it less confusing, but would require another conversion or perhaps a list of minor releases that can run. I'm happy to update this however it best makes sense.

Reply:
This should be an `int`. What if the next release is `5.0.0`?

Reply:
Yeah, I went back and forth on this one, I'll make the change.